parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileReader.java

```diff
@@ -1071,12 +1229,31 @@ public BloomFilterReader getBloomFilterDataReader(BlockMetaData block) {
    */
   public BloomFilter readBloomFilter(ColumnChunkMetaData meta) throws IOException {
     long bloomFilterOffset = meta.getBloomFilterOffset();
+
+    if (0 == bloomFilterOffset) { // TODO Junjie - is there a better way to handle this?
```

Comment:
@ggershinsky, please ensure this is correct and don't keep TODOs in the final code. (We usually don't fix them and they will be there forever.)

Reply:
yep, this works; also, makes sense to me - we should not proceed to read a bloom filter header, if there is no bloom filter in the file; this can be checked via the `bloomFilterOffset` variable - it can't be 0 if a bloom filter is present; and if no bloom filter, it can be 0 only.

I'll remove the TODO comment.